docker-compose.yml

```diff
@@ -1,6 +1,36 @@
-version: '3'
+version: "3"
 
 services:
+  database:
+    image: quay.io/azavea/postgis:3-postgres12.4-slim
+    environment:
+      - POSTGRES_USER=franklin
+      - POSTGRES_PASSWORD=franklin
+      - POSTGRES_DB=franklin
+    healthcheck:
+      test: ["CMD", "pg_isready", "-U", "franklin"]
```

Comment:
```suggestion
      test: pg_isready -U franklin
```

Reply:
Actually, version 3 doesn't support health checks like this. I'd consider using the latest version of the 2.x series.

Reply:
Can you clarify this comment? The docker-compose version 3 reference has a healthcheck that is defined exactly like the one above -- https://docs.docker.com/compose/compose-file/#healthcheck

Reply:
The reason we have the health check is so that we can implement `depends_on` and make other services wait for the database before starting, e.g. https://github.com/azavea/american-water-tank-magic/blob/14d08ad552b4945cca401d9e426150001e8e093c/docker-compose.yml#L38-L40.

You can no longer use `depends_on` with a `condition` in version 3.

Reply:
Ah. 👍 

I removed the healthcheck since it takes awhile for franklin to spool up and I didn't experience any ill effects when this wasn't working. We can revisit if it becomes a problem.